operators/aqua/6.2.1/manifests/aqua-operator.v6.2.1.clusterserviceversion.yaml

```diff
@@ -4,10 +4,16 @@ metadata:
   name: aqua-operator.v6.2.1
   namespace: placeholder
   annotations:
+    # Setting operatorhub.io/ui-metadata-max-k8s-version annotation automatically
+    # The following is an informative annotation to let its users know that this distribution
+    # uses removed APIs in 1.22 and will not work on K8S clusters 1.22+.
+    # Following the findings for this distribution:
+    # this bundle is using APIs which were deprecated and removed in v1.22. More info: https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-22. Migrate the API(s) for CRD: (["aquacsps.operator.aquasec.com" "aquadatabases.operator.aquasec.com" "aquaenforcers.operator.aquasec.com" "aquagateways.operator.aquasec.com" "aquakubeenforcers.operator.aquasec.com" "aquascanners.operator.aquasec.com" "aquaservers.operator.aquasec.com"])
+    operatorhub.io/ui-metadata-max-k8s-version: "1.21"
```

Comment:
It is correct:

```
 operator-sdk bundle validate operators/aqua/6.2.1/  --select-optional suite=operatorframework --optional-values=k8s-version=1.22
WARN[0000] Warning: Value : (aqua-operator.v6.2.1) csv.Spec.minKubeVersion is not informed. It is recommended you provide this information. Otherwise, it would mean that your operator project can be distributed and installed in any cluster version available, which is not necessarily the case for all projects. 
WARN[0000] Warning: Value aqua-operator.v6.2.1: unable to find the resource requests for the container: (aqua-operator). It is recommended to ensure the resource request for CPU and Memory. Be aware that for some clusters configurations it is required to specify requests or limits for those values. Otherwise, the system or quota may reject Pod creation. More info: https://master.sdk.operatorframework.io/docs/best-practices/managing-resources/ 
INFO[0000] All validation tests have completed successfully 
```